app/grandchallenge/reader_studies/templates/reader_studies/display_set_add_images.html

```diff
@@ -0,0 +1,13 @@
+{% load crispy_forms_tags %}
+
+
+<div class="card-body py-1" id="ds-content-{{ display_set }}">
+    <form action="{% url 'reader-studies:add-files-to-display-set' slug=reader_study pk=display_set interface_pk=interface %}" method="post" class="ds-form" data-hx-target="#ds-content-{{ display_set}}" id="form-{{ display_set }}">
```

Comment:
`slug=reader_study` needs to go since the URL path changed. This does not appear to be covered by the tests either.